routing/base/followed_polyline.hpp

```diff
@@ -77,11 +77,44 @@ class FollowedPolyline
   Iter End() const;
   Iter GetIterToIndex(size_t index) const;
 
-private:
+  /// \brief Calculates projection of center of |posRect| to the polyline.
+  /// \param posRect Only projection inside the rect are considered.
```

Comment:
please fix

Reply:
Fixed: `Only projection inside the rect is considered.`